### DISCUSSION TITLE
Arma 3 Creator DLC

### DISCUSSION THREAD

**Discussion by TheShade66** [GameServerManagers/LinuxGSM#4070] [Q&A]
I would like to enable creator dlc on my server, which is normally done trough steam CMD. This is normally done by running the following command:

1-Open SteamCMD

2-Run: app_update 233780 -beta creatordlc " validate +quit

How can I do this using ./arma3server ? or is there another way?

Help is much appreciated.

**Comment by lanerussell** [comment]
Over a year on, I'm guessing this still isn't supported?

**Reply by Timadi** [reply]
If you go Into LGSM/Modules and open "core_steamcmd.sh" on line 178 you can change the Branch from "Public" to "creatordlc"

**Reply by h3o66** [reply]
please do not change the lgsm source when there is already a build in solution:
https://docs.linuxgsm.com/steamcmd/branch

**Comment by TheShade66** [comment]
I switched to hosting my server via Windows, because Linux arma 3 server has A LOT of workarounds needed / symlinks setting / worse load performance. 

I would recommend running either on native windows or in a virtual box, I switched after a year of issues (antistasi took 10-15 mins to initilaze, unfixable. On windows it takes 3 min max)

I now use this tool: https://github.com/Foxlider/FASTER
